packages/patternfly-4/react-core/src/components/Switch/Switch.tsx

```diff
@@ -87,3 +86,7 @@ export class Switch extends React.Component<SwitchProps> {
     );
   }
 }
+
+const SwitchWithOuiaContext = withOuiaContext(Switch);
+
+export { SwitchWithOuiaContext as Switch };
```

Comment:
React-docgen is still unphased when grabbing the props for Switch. This is good.